api/v1alpha1/workspace_validation.go

```diff
@@ -45,7 +45,7 @@ func (w *Workspace) Validate(ctx context.Context) (errs *apis.FieldError) {
 		errs = errs.Also(w.validateCreate().ViaField("spec"))
 		if w.Inference != nil {
```

Comment:
We need to ensure either one of tunning or inference spec should be set, not both.

Reply:
func (w *Workspace) validateCreate() (errs *apis.FieldError) {
	if w.Inference == nil && w.Tuning == nil {
		errs = errs.Also(apis.ErrGeneric("Either Inference or Tuning must be specified, not neither", ""))
	}
	if w.Inference != nil && w.Tuning != nil {
		errs = errs.Also(apis.ErrGeneric("Either Inference or Tuning must be specified, but not both", ""))
	}
	return errs
}